src/components/Input/Input.js

```diff
@@ -377,8 +381,8 @@ Input.propTypes = {
 Input.defaultProps = {
   children: null,
   element: 'input',
-  prefix: null,
-  suffix: null,
+  renderPrefix: null,
+  renderSuffix: null,
```

Comment:
Did you update the prop names in the SearchInput component as well?

Reply:
Yes.